test/coverband/configuration_test.rb

```diff
@@ -86,4 +86,23 @@ def setup
       end
     end
   end
+
+  test 'use_oneshot_lines_coverage' do
+    Coverband::Collectors::Coverage.instance.reset_instance
+    refute Coverband.configuration.use_oneshot_lines_coverage
+
+    Coverband.configuration.stubs(:one_shot_coverage_implemented_in_ruby_version?).returns(true)
```

Comment:
Not for this PR, but I think we should try to avoid stubbing in the class under test. I think with some refactoring, we could stick all of the ruby version checks in another class and inject our own class double within our tests.